src/components/vm/disks/vmDisksCard.tsx

```diff
@@ -269,7 +220,49 @@ export const VmDisksCard = ({
                                 })
                                 } />
         });
-        return { columns, props: { key: idPrefixRow } };
+
+        const Description = ({ term, children } : { term: string, children: React.ReactNode }) => (
+            <DescriptionListGroup>
+                <DescriptionListTerm>{term}</DescriptionListTerm>
+                <DescriptionListDescription>{children}</DescriptionListDescription>
+            </DescriptionListGroup>
+        );
+
+        const expandedContent = (
+            <DescriptionList isHorizontal>
+                { !isNaN(Number(disk.used)) &&
+                    <Description term={_("Used on host")}>
+                        <StorageUnit value={disk.used} id={`${idPrefixRow}-used`} key={`${idPrefixRow}-used`} />
+                    </Description>
+                }
+                <Description term={_("Bus")}>
+                    {disk.bus}
+                </Description>
+                <Description term={_("Access")}>
+                    <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }} id={`${idPrefixRow}-access`}>
+                        <FlexItem>{ disk.readonly ? _("Read-only") : disk.shareable ? _("Concurrently writeable") : _("Writeable") }</FlexItem>
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2324-dfb0aba1-20250916-120800-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)